modules_for_testing/bank_module.py

```diff
@@ -0,0 +1,113 @@
+"""BANK MODULE"""
+
+import logging
+from decimal import Decimal
+
+formatter = logging.Formatter(
+    '\n%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+handler = logging.StreamHandler()
+handler.setFormatter(formatter)
+
+logger = logging.getLogger(__name__)
+logger.addHandler(handler)
+logger.setLevel(logging.INFO)
```

Comment:
You can use 1 logger across the application, e.g.: https://stackoverflow.com/questions/75489656/how-to-use-a-single-python-logging-object-throughout-the-python-application

Reply:
Reworked project, so now it has separate module with logging setup and other modules import it